src/main/scala/org/globalforestwatch/summarystats/SummaryAnalysisFactory.scala

```diff
@@ -1,6 +1,8 @@
 package org.globalforestwatch.summarystats
 
 import cats.data.NonEmptyList
+import geotrellis.spark.SpatialKey
+import geotrellis.spark.partition.SpacePartitioner
```

Comment:
Codacy found an issue: [Imports should be sorted alphabetically](https://app.codacy.com/manual/thomas-maschler/gfw_forest_loss_geotrellis/pullRequest?prid=5902460)